cmake/Utils.cmake

```diff
@@ -7,6 +7,12 @@ function(add_msvc_runtime_flag lib)
     else()
       target_compile_options(${lib} PRIVATE /MT)
     endif()
+  else()
+    if(${CMAKE_BUILD_TYPE} MATCHES "Debug")
```

Comment:
See #1995 

It interferes in creating VS Solutions that support multiple configurations.

Reply:
See #1563

Reply:
https://stackoverflow.com/questions/24460486/cmake-build-type-is-not-being-used-in-cmakelists-txt

Reply:
Looks like we can get rid of this else which I added since the that is the default anyways... earlier 
-DONNX_USE_MSVC_STATIC_RUNTIME was always set to ON state... now we have an option to set this to OFF...

Reply:
Yes, just need to come up with a way without if/else clause

---
In reply to: [346084942](https://github.com/onnx/onnx/pull/2452#discussion_r346084942) [](ancestors = 346084942)